.github/workflows/release.yml

```diff
@@ -26,7 +27,8 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y autoconf automake build-essential cmake gettext gperf libtool-bin locales ninja-build pkg-config unzip
+          sudo apt-get install -y autoconf automake build-essential cmake gcc-11 gettext gperf libtool-bin locales ninja-build pkg-config unzip
+          sudo update-alternatives --install /usr/bin/cc cc /usr/bin/gcc-11 100
```

Comment:
I would need to double check the `update-alternatives` man page to make sure this is doing the right thing.

It's probably clearer to just prepend `CC=gcc-11` to the make commands

Reply:
I'm almost certain `update-alternatives` would have done the right thing - I tried it on my machine before pushing.

For some reason I thought setting env variables before running a program didn't work on github actions, but I might be confusing github with another CI provider.

Thanks for the review :)